bifrost.go

```diff
@@ -85,6 +88,28 @@ func NewRainbowBridge(bc *BridgeConfig) (RainbowBridge, error) {
 	}
 
 }
+func newPinataIPFS(g *BridgeConfig) (rainbowBridge, error) {
+	
+	if g.PinataJWT == "" {
+			return nil, &errors.BifrostError{
+				Err:       fmt.Errorf("jwt is required for authentication"),
```

Comment:
I understand that we'll be changing this soon enough to allow other authentication patterns other than JWT, however, the error would have been best written as `pinata JWT is required` this way they get to make sense of which bridge config is missing.